PiwikTracker.php

```diff
@@ -1288,6 +1288,25 @@ public function setRequestTimeout($timeout)
     }
 
     /**
+     * If a proxy is needed to look up the address of the Piwik site, set it with this
+     * @param $proxy
+     * @param int $proxyPort
```

Comment:
Hi, sorry for the late answer, I have been on parental leave for several months on I haven't checked any work related accounts :)
I have added the descriptions and the code indentations as requested.